client/app/components/SkipToContent/index.jsx

```diff
@@ -0,0 +1,15 @@
+// @flow
+import React from 'react';
+import css from './SkipToContent.scss';
+
+export type Props = {
+  id?: string,
```

Comment:
I would make this prop required.

Reply:
So this issue is responsible for the following Flow error that's showing up in the CircleCI build:

```
Cannot coerce `props.id` to string because undefined [1] should not be coerced.
 

 
   app/components/SkipToContent/index.jsx:13:53
 
   13|   return <a className={css.skipToContent} href={`#${props.id}`}>Skip to main content</a>;
 
                                                           ^^^^^^^^
 

 
References:
 
   app/components/SkipToContent/index.jsx:7:8
 
    7|   id?: string,
 
              ^^^^^^ [1]
 
```